### ISSUE TITLE
Specify custom value for enum string ?

### ISSUE THREAD

**Issue by SkYNewZ** [abice/go-enum#136]
Is it possible to specify custom value for enum string ? Like this for example:

```go
/* ENUM(
PROD = "https://prod",
DEV = "http://dev"
)
*/
type Environment string
```

Which could generate this
```go
const (
	PROD Environment = "https://prod"
	DEV Environment = "http://dev"
)
```

**Comment by abice** [comment]
I'm pretty early on in my string support, and there might be a need for yet another CLI flag... But it's probably doable... Not sure when I'll have the time to look at it again. I just threw up a PR for adding sql facing ints with the string type enum... Hopefully I can get them both to work (not at the same time mind you, they'd be mutually exclusive features).

**Comment by SkYNewZ** [comment]
> I'm pretty early on in my string support, and there might be a need for yet another CLI flag... But it's probably doable... Not sure when I'll have the time to look at it again. I just threw up a PR for adding sql facing ints with the string type enum... Hopefully I can get them both to work (not at the same time mind you, they'd be mutually exclusive features).

Great! Thanks for your answer and no worries it's not urgent 🙂

**Comment by abice** [comment]
Whoops!  Forgot to mark this as resolved in the PR that took care of it!

Fixed in #165 

Let me know if you have any issues!

**Comment by SkYNewZ** [comment]
Awesome! Thank you
